### ISSUE TITLE
Integrating with RNN

### ISSUE THREAD

**Issue by FRizzonelli** [ianlin/react-native-callkit#16]
Hi there! Great work on the lib!

I'm not an iOS dev and I'm having some problem to integrate this with [react-native-navigation](https://github.com/wix/react-native-navigation)
For the other lib, I need to do this 
```
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  // Initialise RNCallKit
  RNCallKit *rncallkit = [[RNCallKit alloc] init];
  
  NSURL *jsCodeLocation;

  jsCodeLocation = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index.ios" fallbackResource:nil];

  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
  self.window.backgroundColor = [UIColor whiteColor];
  
  [[RCCManager sharedInstance] initBridgeWithBundleURL:jsCodeLocation launchOptions:launchOptions];
  
  self.oneSignal = [[RCTOneSignal alloc] initWithLaunchOptions:launchOptions
                                                         appId:@"oneSignalId"];

  /*
  RCTRootView *rootView = [[RCTRootView alloc] initWithBundleURL:jsCodeLocation
                                                      moduleName:@"myApp"
                                               initialProperties:nil
                                                   launchOptions:launchOptions];
  rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];

  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
  UIViewController *rootViewController = [UIViewController new];
  rootViewController.view = rootView;
  self.window.rootViewController = rootViewController;
  [self.window makeKeyAndVisible];
  */
  return YES;
}
```

I'm trying to init your lib and import it as a NativeModule, but on react-native I receieve the error `Native Modules cannot be null`

Any idea on how to properly link your lib as Native Module?

**Comment by ianlin** [comment]
Thanks!

From your code it looks like you didn't init `React Bridge` and `React Root View` as described [here][1]

[1]: https://github.com/ianlin/react-native-callkit#--change-the-way-you-initialise-react-root-view

**Comment by ianlin** [comment]
Yeah you can do it and if you do it you have to remove `RNCallKit *rncallkit = [[RNCallKit alloc] init];` from `AppDelegate.m` otherwise `RNCallKit` would be initialised twice.

The reason why initialising `RNCallKit` in `AppDelegate.m`  is that I want `RNCallKit` to be loaded as fast as possible. It could be unnecessary now, I'll have a look, cheers!

**Comment by FRizzonelli** [comment]
Perfect!

Thank you :) Cheers!

**Comment by ianlin** [comment]
Hey just released `1.3.0` including remove the comment of `RCT_EXPORT_MODULE()`

**Comment by platonish** [comment]
@ianlin / anyone else,  RNN v2 now has a different setup [(link)](https://wix.github.io/react-native-navigation/#/docs/Installing?id=ios). I'm not an iOS developer and am not able to use this module with RNN v2. Any help would be much appreciated!
